### DISCUSSION TITLE
Can we have an Object Level Security Editor similar to the Perspective Editor

### DISCUSSION THREAD

**Discussion by samaguire** [TabularEditor/TabularEditor3#1029] [Ideas]
I'd love it if we had an editor like the Perspective Editor but for setting table/column object level security. The existing way of doing this is clunky if you need to do this over a number of columns for more than a couple of roles.

**Comment by mlonsk** [comment]
Hi @samaguire 
Yes, agreed that would be a lot easier. It is a great idea and we have it on our backlog, but I do not currently have a timeline for it. 
We would also want to incorporate RLS into the same role editor experience.  
Thank you for the suggestion.
